### ISSUE TITLE
Blast Furnace still eating ingots

### ISSUE THREAD

**Issue by Pelirow** [AnonymousProductions/MineFantasy2#26]
It appears your "check only one block" aspect of the blast furnace has some issues. It might be checking the top chamber instead of the bottom one.

In my two-chamber blast furnace, if I try to smelt just one unrefined ingot and it travels down to the bottom of the two chambers, when a smelting cycle is done, the unrefined ingot is eaten. Tested twice with unrefined red steel.

**Comment by AnonymousProductions** [comment]
what? it runs a check from +1 to +16 down to up

It's ment to fall to the bottom if thats what you mean, but if you mean the closest one to the heater is ignored than thats a problem

**Comment by Pelirow** [comment]
Did further testing with 3 and 4 chamber furnaces as well. Multiple issues.

If there's nothing in the top chamber, everything below gets eaten. If every chamber is full, it works as intended. 

I tested the four chamber blast furnace, I put in 2 unrefined and 2 coal, they worked down to the bottom two chambers, leaving the top two empty. The furnace would not function at all until the topmost chamber was broken, then the heater would work with the remaining three chambers, but still eats the two unrefined ingots.

I filled the 4-chamber furnace with 4 unrefined ingots and  4 coal and it would start work then stop, breaking the top chamber would again start up, and the 3 remaining chambers would work as intended and output their ingots in the crucible below.
